src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -197,9 +198,9 @@ void AppendOrdering(IRestRequest request, string orderBy, bool ascending)
             request.AddParameter("direction", ascending ? "asc" : "desc", ParameterType.GetOrPost);
         }
 
-        void AppendPaging(IRestRequest request, int pageIndex)
+        void AppendPaging(IRestRequest request, int pageSize)
```

Comment:
It has a default value in SC (same as here, 50 messages). Do you say we should just rely on the SC default here?